modin/pandas/general.py

```diff
@@ -370,12 +370,32 @@ def unique(values):  # noqa: PR01, RT01, D200
     return Series(values).unique()
 
 
-@_inherit_docstrings(pandas.value_counts, apilink="pandas.value_counts")
+# adding docstring since pandas docs doesn't have web section for this function
```

Comment:
Please submit a question on this in pandas GH. Same is for other related functions.

Reply:
Ok, question was submitted https://github.com/pandas-dev/pandas/issues/46749.

Reply:
pandas answered that these functions are considered to be removed in the future releases, so i think we can leave docstrings until these functions are removed from pandas api to drop it in Modin too.